src/main/java/FileClass.java

```diff
@@ -0,0 +1,46 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class FileClass {
+
+    public static void createFile(String filename) {
+        File f = new File(filename);
+        System.out.println("full path: " + f.getAbsolutePath());
+        System.out.println("file exists?: " + f.exists());
+        System.out.println("is Directory?: " + f.isDirectory());
```

Comment:
Perhaps the user does not need to know about the file path validation process? I would suggest these be used more as debugging statements!